frida_mode/src/lib/lib.c

```diff
@@ -39,7 +39,6 @@ typedef struct {
 static guint64 text_base = 0;
 static guint64 text_limit = 0;
 
-  #ifdef GUM_16_6_PLUS
```

Comment:
So we drop support for older frida versions?? Makes a lot of sense, just double checking